client/components/main/style.scss

```diff
@@ -4,11 +4,14 @@
 	z-index: z-index( 'root', '.main' );
 
 	// Themes is a great example of using all this new space ;)
-	&.themes,
-	&.sites {
+	&.themes {
 		max-width: 100%;
 	}
 
+	&.sites {
+		max-width: 320px;
```

Comment:
Makes sense for the selector itself, but leads to some undesirable (in my opinion) wrapping for the header text:

![image](https://cloud.githubusercontent.com/assets/1779930/13332657/1defc156-dbd1-11e5-99ca-ddf39fb69116.png)